go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/go-logr/zapr v1.2.0
 	github.com/go-test/deep v1.0.8
 	github.com/golang/mock v1.6.1-0.20220512030613-73266f9366fc
+	github.com/golang/protobuf v1.5.2
```

Comment:
This package is deprecated. Use `google.golang.org/protobuf` instead.

Reply:
Yep IDE auto-import that I missed, fixed